package/MDAnalysis/core/AtomGroup.py

```diff
@@ -671,6 +749,7 @@ def force(self, vals):
         except (AttributeError, NoDataError):
             raise NoDataError("Timestep does not contain forces")
 
+    @deprecate(message="{}; use `position` property instead".format(_FIFTEEN_DEPRECATION))
     def centroid(self):
```

Comment:
I'm not in favour of centroid per se, but I'd rather keep things focused rather than throwing in changes we'd like.